quadrat/sass/blocks/_list.scss

```diff
@@ -0,0 +1,4 @@
+ul ul,
+ol ul {
```

Comment:
Based on conversation and comps I believe this should be:
```
ul li,
ol li {
```

Reply:
`list-style-type` is a property of `ol` and `ul` not `li`.

Reply:
I [beg to differ](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-type).

> Only a few elements (<li> and <summary>) have a default value of display: list-item. However, the list-style-type property may be applied to any element whose display value is set to list-item. Moreover, because this property is inherited, it can be set on a parent element (commonly <ol> or <ul>) to make it apply to all list items.

Reply:
However, I see my suggestion was wrong, I think only this is necessary (so that we don't style ol's too)
```
ul li {
  sub-list-style-type: disc;
}
```
Otherwise I continue to see the default stylings for anything other than immediate children of the ul.

Reply:
Also I clearly don't know what I'm doing because `ul ul {` also work because of course you know. :P

However, the branch wasn't working correctly, and it must have been caused by something else.